src/main/windows/webtorrent.js

```diff
@@ -26,7 +25,8 @@ function init () {
     title: 'webtorrent-hidden-window',
     useContentSize: true,
     webPreferences: {
-      nodeIntegration: true
+      nodeIntegration: true,
+      backgroundThrottling: false // do not throttle animations/timers when page is background
```

Comment:
I'm not sure this was in effect before, do we actually want to disable background throttling?

Reply:
Given that it was not in inside `webPreferences` I assume it was in it's default state: `true`. So I wouldn't change it here 😅